docs/tutorials/geomesa-blobstore-exif.rst

```diff
@@ -0,0 +1,170 @@
+the EXIF FileHandler to an existing GeoServer with the BlobStore web servlet. See instructions above for
+setting up the BlobStore in the prerequisites.
+
+To deploy the EXIF FileHandler if you are using Tomcat run:
+
+.. code-block:: bash
+
+    $ tar -xzvf \
+      geomesa-$VERSION/dist/handlers/geomesa-blobstore-exif-handler-$VERSION.jar \
+      -C /path/to/tomcat/webapps/geoserver/WEB-INF/lib/
+
+To deploy the EXIF FileHandler if you are using GeoServer's built in Jetty web server:
+
+.. code-block:: bash
+
+    $ tar -xzvf \
+      geomesa-$VERSION/dist/handlers/geomesa-blobstore-exif-handler-$VERSION.jar\
+      -C /path/to/geoserver/webapps/geoserver/WEB-INF/lib/
+
+Now the BlobStore is able to read image files with EXIF metadata headers.
+
+
+Using the EXIF Handler
+----------------------
+In order to use our new handler, we will need to get some geotagged images.
+Below is a partial Python script that uses the Flickr API to grab some geotagged images taken around Monument Valley.
+Using the Flickr API is beyond the scope of this tutorial, however the Python code snippet is provided for those
+interested in obtaining a test data set. The below script will download 100 images to the user's home directory.
+
+.. code-block:: python
+
+    import flickrapi
+    import requests
+    import itertools
+    api_key = u'Some API Key'
+    secret  = u'some secret'
+
+    flickr = flickrapi.FlickrAPI(api_key, secret)
+    flickr.authenticate_console(perms='read')
+    photos = flickr.photos_search(lat='37.0000', lon='-110.1700', radius='10', safe_search='1', extras='url_o')
+
+    urls = itertools.islice((p.attrib['url_o'] for p in photos[0] if 'url_o' in p.keys()), 100)
+
+    for i, url in enumerate(urls):
+        with open('~/flickr{0:03d}.jpg'.format(i), 'wb') as fd:
+            r = requests.get(url)
+            for chunk in r.iter_content(1024):
+                fd.write(chunk)
+
+
+
+First we need to register a BlobStore using the following command. Note that the parameters for user/password/zookeepers/instanceId need to be modified to suite your environment:
+
+.. code-block:: bash
+
+    $ curl -d 'instanceId=myCloud' -d 'zookeepers=zoo1,zoo2,zoo3' -d 'tableName=myblobstore' -d 'user=user' -d 'password=password' http://localhost:8080/geoserver/geomesa/blobstore/ds/myblobstore
+
+To ingest the files, we can write a simple bash loop to use cURL on each file to ingest to the BlobStore.
+
+.. code-block:: bash
+
+    $ for f in *.jpg; do curl -X POST -F file=@$f http://localhost:8080/geoserver/geomesa/blobstore/blob/myblobstore ; done
+
+The files have now been ingested.
+
+
+Register Index table in GeoServer
+---------------------------------
+
+The BlobStore index DataStore is an ordinary GeoTools DataStore, so the registration in GeoServer is no different than for other DataStores.
+For an example of registering a DataStore in GeoServer please follow these :doc:`/user/geoserver.rst'.
```

Comment:
[picky RST] Both quotes around the doc reference need to be backticks (LaTeX habits can be hard to break!), and you should omit the .rst.